docs/web_reference.rst

```diff
@@ -1049,6 +1049,10 @@ and :ref:`aiohttp-web-signals` handlers::
       May be ``None`` if server and client protocols are
       not overlapping.
 
+   .. method:: get_extra_info(name, default=None)
+
+      Reads extra info from writer's transport
```

Comment:
It'd be useful to see more details here. Like, what's that info? Why is it extra? What's non-extra? What's the format? Where's the data coming from? What is it related to?

Saying that it's some arbitrary data coming from the transport layer is not really helpful and is too generic.

Oh, and I see a missing trailing period that should be used to terminate sentences.

Reply:
Well, this is just proxying the call to asyncio, right?
So, both this one and the original it was copied from should be changed to reference `asyncio.BaseTransport.get_extra_info()`. Then users can just look at the Python documentation for details.

Reply:
I've added reference for python docs just in case

Reply:
We use Intersphinx, so my suggestion below is a link to the docs.

Reply:
resolved